### PR TITLE
pipwin: add page

### PR DESCRIPTION
**Pull request by flydeoo**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/pipwin.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# pipwin
+
+> Complementary tool for pip on Windows.
+> Pipwin installs unofficial python package binaries for windows.
```
Comment (navarroaxel) [inline pages/common/pipwin.md:4]: I think we can simplify the command description
```suggestion
> Complementary tool to install unofficial Python package binaries on Windows.
```

File: pages/common/pipwin.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Search packages:
+
+`pipwin search {{partial_name/name}}`
+
+- Install a package:
+
+`pipwin install {{package_name}}`
+
+- Uninstall a package(Can directly use pip for this):
```
Comment (navarroaxel) [inline pages/common/pipwin.md:19]:
```suggestion
- Uninstall a package (can directly use pip for this):
```

File: pages/common/pipwin.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`pipwin uninstall {{package_name}}`
+
+- Download only (to special directory if provided):
```
Comment (navarroaxel) [inline pages/common/pipwin.md:23]:
```suggestion
- Download only (to a specific directory if provided):
```

File: pages/common/pipwin.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`pipwin download -d {{path/to/directory}} {{package_name}}`
```
Comment (navarroaxel) [inline pages/common/pipwin.md:25]: We prefer long arguments if available
```suggestion
`pipwin download --dest={{path/to/directory}} {{package_name}}`
```

File: pages/common/pipwin.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- install from requirements file:
+
+`pipwin install -r {{path/to/requirements.txt}}`
```
Comment (navarroaxel) [inline pages/common/pipwin.md:29]:
```suggestion
`pipwin install --file={{path/to/requirements.txt}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @flydeoo, welcome and thanks for your submission.
If this is a command for the Windows platform, the page should be in the `windows/` folder.

File: pages/common/pipwin.md (hunk 5 above)
Comment (navarroaxel) [reply pages/common/pipwin.md:29]: @flydeoo you resolved this conversation but you didn't change the example.

**Comment by flydeoo** [comment]
Hi @navarroaxel 
Thanks for the tips, I will close this PR and send another one with the proper changes.

**Comment by navarroaxel** [comment]
> Hi @navarroaxel Thanks for the tips, I will close this PR and send another one with the proper changes.

@flydeoo No, please, the PR is for reviews, suggestions and changes. If you create a new PR we can't track the suggestions and changes. Please continue your work here.

**Comment by navarroaxel** [comment]
@flydeoo it's pending move the file to the `windows/` directory.

**Review by mfrw — APPROVED** [review]
LGTM.
Thank you @flydeoo for your contribution.

File: pages/windows/pipwin.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# pipwin
+
+> Complementary tool to install unofficial Python package binaries on Windows.
```
Comment (CleanMachine1) [inline pages/windows/pipwin.md:3]:
```suggestion
> A Tool to install unofficial Python package binaries on Windows.
```

File: pages/windows/pipwin.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://github.com/lepisma/pipwin>.
+
+- List all available packages for download:
+
+`pipwin list`
+
+- Search packages:
+
+`pipwin search {{partial_name/name}}`
```
Comment (CleanMachine1) [inline pages/windows/pipwin.md:12]:
```suggestion
`pipwin search {{partial_name|name}}`
```

What is a partial name, could you provide an example of both.

File: pages/windows/pipwin.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install a package:
+
+`pipwin install {{package_name}}`
+
+- Uninstall a package (can directly use pip for this):
```
Comment (CleanMachine1) [inline pages/windows/pipwin.md:18]:
```suggestion
- Uninstall a package:
```

File: pages/windows/pipwin.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`pipwin uninstall {{package_name}}`
+
+- Download only (to a specific directory if provided):
```
Comment (CleanMachine1) [inline pages/windows/pipwin.md:22]:
```suggestion
- Download a package to a specific directory:
```

File: pages/windows/pipwin.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+`pipwin download --dest={{path/to/directory}} {{package_name}}`
```
Comment (CleanMachine1) [inline pages/windows/pipwin.md:24]: Does this have the ability to use the following suggestion?
```suggestion
`pipwin download --dest {{path/to/directory}} {{package_name}}`
```

File: pages/windows/pipwin.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+- Install from requirements file:
```
Comment (CleanMachine1) [inline pages/windows/pipwin.md:26]:
```suggestion
- Install packages according to `requirements.txt`:
```

File: pages/windows/pipwin.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+`pipwin install --file={{path/to/requirements.txt}}`
```
Comment (CleanMachine1) [inline pages/windows/pipwin.md:28]: Same applies here

File: pages/windows/pipwin.md (hunk 7 above)
Comment (flydeoo) [reply pages/windows/pipwin.md:12]: yes, sure.
Due to information provided in developer github page, I've used "partial name" :
 
 `pipwin search plot`
 
 Did you mean any of these ?

 * bqplot
 * ggplot
 * hvplot
 * matplotlib
 * matplotlib-inline
 * plotly
 * plotnine
 * scikits.vectorplot

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**